app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java

```diff
@@ -115,58 +152,78 @@ protected void onCreate(Bundle savedInstanceState) {
             setImageFromPath(path);
         }
 
-        if (!checkColumns())
-        {
-            _fragmentColumns = new FragmentColumns();
+        if (savedInstanceState != null) {
+            _columns = savedInstanceState.getInt(_COUNT_COLUMNS_TAG, 0);
+            _rows = savedInstanceState.getInt(_COUNT_ROWS_TAG, 0);
+            _isSelectedColumns = savedInstanceState
+                    .getInt(_IS_SELECTED_COLUMNS_TAG, 0) != 0;
+            byte[] byteArray = savedInstanceState.getByteArray(_BMP_CURRENT_IMAGE_TAG);
+            if (byteArray != null)
+                _bmpCurrentImage =
```

Comment:
Please check [`if`](https://github.com/nonoteam/NonoConverter/blob/master/CodingStandard.md#use-standard-brace-style) in CodingStandard.md